.drone.yml

```diff
@@ -0,0 +1,84 @@
+clone:
+  git:
+    image: registry.fpfis.eu/fpfis/httpd-php:${PHP_VERSION}-ci
+    when:
+      event: [ push, pull_request, tag ]
+  git:
+    image: registry.fpfis.eu/drone-plugins/git
+    when:
+      event: [ deployment ]
+
+workspace:
+  base: /test
+  path: oe_starter_content
+
+services:
+  web:
+    image: registry.fpfis.eu/fpfis/httpd-php:${PHP_VERSION}-ci
+    environment:
+      - DOCUMENT_ROOT=/test/oe_starter_content
+      - COMPOSERVER=--2
+  mysql:
+    image: registry.fpfis.eu/fpfis/sql:percona-5.7
+    command: --innodb-log-file-size=1G --max_allowed_packet=1G --innodb-buffer-pool-size=512M --wait_timeout=3000 --net_write_timeout=3000 --log_error_verbosity=3
+    environment:
+      - MYSQL_ALLOW_EMPTY_PASSWORD=yes
+      - MYSQL_DATABASE=drupal
+
+pipeline:
+  composer-install-highest:
+    group: prepare
+    image: registry.fpfis.eu/fpfis/httpd-php:${PHP_VERSION}-ci
+    volumes:
+      - /cache:/cache
+    commands:
+      - composer self-update --2
+      - composer install --ansi --no-suggest --no-progress
+    when:
+      matrix:
+        COMPOSER_BOUNDARY: highest
+
+  composer-update-lowest:
+    group: prepare
+    image: registry.fpfis.eu/fpfis/httpd-php:${PHP_VERSION}-ci
+    volumes:
+      - /cache:/cache
+    commands:
+      - composer self-update --2
+      - composer update --prefer-lowest --prefer-stable --ansi --no-progress
```

Comment:
Same as above.

Reply:
Should I remove `--no-progress`? There's no `--no-suggest` on this line.

Reply:
@GilNovacomm --no-progress stays, I found this document helpful https://getcomposer.org/upgrade/UPGRADE-2.0.md check it out

Reply:
It was my bad, I read suggest instead of progress 😞